lib/rules/prefer-output-null.js

```diff
@@ -45,10 +45,9 @@ module.exports = {
             */
             function getTestInfo (key) {
               if (test.type === 'ObjectExpression') {
-                const res = test.properties.filter(item => item.key.name === key);
-                return res.length > 0 ? res[res.length - 1] : null;
+                return test.properties.find(item => item.key.name === key);
               }
-              return key === 'code' ? test : null;
+              return null;
```

Comment:
No functional change here. Just removing some dead codepaths that can't be hit for invalid test cases.